pkg/console/tlsconfig/tlsconfig_test.go

```diff
@@ -143,6 +143,25 @@ var _ = Describe("TlsConfig", func() {
 		Expect(config.MinVersion).To(Equal(uint16(tls.VersionTLS12)))
 	})
 
+	It("should use no ciphers if ciphers is an empty array", func() {
```

Comment:
Can you explain the use case for this? IMO it makes more sense to use defaults if the array was empty.

Reply:
In case we want to only use TLS 1.3 ciphers, the array can be left empty.
This is to make it compatible with OpenShift's library: `github.com/openshift/library-go/pkg/crypto`. I'm using it in my PR on SSP operator: 
https://github.com/kubevirt/ssp-operator/pull/1046/files#diff-2e1a8219314708dfe8f1f4d851b52cf3c444ec161e4261ce2f8e8f89a12a8b6eR386

The function `crypto.OpenSSLToIANACipherSuites()` ignores any TLS 1.3 ciphers, so if we use `Modern` TLS profile from the OpenShift API, then it would return an empty slice. In that case, we don't want to use default ciphers.

Reply:
If TLS 1.3 ciphers are not configurable how does it make a difference if an empty or a nil slice is used?

Reply:
- `nil` = use default ciphers + TLS 1.3 ciphers.
- `[]` = use only TLS 1.3 ciphers.

This is the relevant function that sets ciphers for a TLS connection: https://github.com/golang/go/blob/cb4eee693c382bea4222f20837e26501d40ed892/src/crypto/tls/common.go#L1013-L1024

The `defaultCipherSuites` is populated here:
https://github.com/golang/go/blob/cb4eee693c382bea4222f20837e26501d40ed892/src/crypto/tls/cipher_suites.go#L352-L364

And I also guess that the TLS 1.3 ciphers are only used if the client wants to use TLS 1.3. So if we pass `[]` into the ciphers, the client cannot use lower TLS version.

I wrote this change, so we can write an empty config file `{}`, and it would mean "Use default TLS configuration",
but also a config file:
```yaml
ciphers: []
minTLSVersion: "VersionTLS13"
```
To specify that we don't want any default ciphers.

Reply:
And with only `minTLSVersion: "VersionTLS13"` a client would still be able to use ciphers of lower TLS versions?

Reply:
I'm not sure. I can look more into the source code, if it's important.